package.json

```diff
@@ -23,19 +23,19 @@
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/contact-info": "1.2.15",
     "@bcrs-shared-components/corp-type-module": "1.0.11",
-    "@bcrs-shared-components/correct-name": "1.0.9",
+    "@bcrs-shared-components/correct-name": "1.0.33",
```

Comment:
Since I'm using this shared component, I figured I'd better upgrade to the latest version first.